From dcbea1981a0eb81857043509043f75873e2396e6 Mon Sep 17 00:00:00 2001
From: Ludovic Brenta <ludovic@ludovic-brenta.org>
Date: Wed, 8 Apr 2020 14:44:15 +0200
Subject: [PATCH 2/3] Always ignore errors from gprclean.

---
 build/gnat/makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/build/gnat/makefile b/build/gnat/makefile
index 779a091..e6d94b7 100644
--- a/build/gnat/makefile
+++ b/build/gnat/makefile
@@ -45,7 +45,7 @@ $(LIB_NAME): sdlada.gpr $(GEN_KB)
 .PHONY: clean-lib
 
 clean-lib:
-	$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) \
+	-$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) \
 		-Psdlada.gpr
 
 
@@ -68,7 +68,7 @@ clean-kb:
 
 clean-tools:
 
-	$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptools.gpr
+	-$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptools.gpr
 
 #####################################################################################
 # Tests
@@ -90,8 +90,8 @@ gen/$(SDL_MODE)/test/libtestmaths.so: $(SRC_DIR)/test/maths/maths.ad[sb] $(SRC_D
 .PHONY: clean-tests
 
 clean-tests:
-	$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptests.gpr
-	$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptest_maths_build.gpr
+	-$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptests.gpr
+	-$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Ptest_maths_build.gpr
 
 #####################################################################################
 # Unit tests
@@ -105,7 +105,7 @@ build_unit_tests/unit_tests:
 .PHONY: clean-unit_tests
 
 clean-unit_tests:
-	-$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Punit_tests.gpr
+	--$(GPRCLEAN) -XSDL_MODE=$(SDL_MODE) -XSDL_PLATFORM=$(SDL_PLATFORM) -Punit_tests.gpr
 
 #####################################################################################
 .PHONY: install
-- 
2.25.1

